2020/index.html

```diff
@@ -260,6 +260,36 @@ <h3 class="card-title text-white">Capyweb</h3>
             </div>
         </div>
 
+        <div class="project-4 section section-dark">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-12 ml-auto mr-auto text-center">
+                        <h2 class="title">Nossos Parceiros</h2>
+                        <div class="col-md-4">
+                            <div class="card card-pricing">
+                                <div class="card-body">
+                                    <div class="card-icon">
+                                        <img src="assets/img/partners/sticker-mule-logo-light-bg-stacked.png" alt="Sticker Mule">
+                                    </div>
+                                    <h3 class="card-title text-white">Sticker Mule</h3>
+                                    <p class="card-description">
+                                    </p>
+                                    <div class="card-footer">
+                                        <a href="http://www.stickermule.com/supports/darkmiratour" target="_blank" class="btn btn-success btn-round card-link">Conheça a Sticker Mule</a>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="space-top"></div>
```

Comment:
I think this is not necessary and the div class row below too

Reply:
Done